plat/kvm/x86/efi_post.c

```diff
@@ -47,6 +49,7 @@ static inline void unmask_8259_pic(void)
 /* UEFI enables the LAPIC Timer to run periodic routines, usually at 10KHz */
 static inline void lapic_timer_disable(void)
 {
+#if CONFIG_LIBUKINTCTLR_APIC
```

Comment:
Hmm, weird how I did not bump into this error. But it makes sense now :+1:. Problem is, this means that the LAPIC Timer would only get disabled if the APIC driver is enabled (which seems to happen implicitly for SMP builds only), which is not ok. Let's follow the PIC example (see the PIC macros defined above) and declare duplicate macros for `APIC_MSR_BASE` and `APIC_BASE_EN` in this file and add a comment with a TODO to let the reader know this should be removed once we handle the LAPIC Timer in our APIC driver.

Reply:
> which seems to happen implicitly for SMP builds only

Ahh.. now I know I did not bump into this... I always have SMP enabled... :laughing: